dingus/src/glrt.ts

```diff
@@ -14,7 +14,11 @@
 declare function require(name: string): any;
 
 const eye = require('eye-vector');
-const mat4 = require('gl-mat4');
+const mat4 = require('gl-matrix').mat4;
+const mat3 = require('gl-matrix').mat3;
+const vec4 = require('gl-matrix').vec4;
+const vec3 = require('gl-matrix').vec3;
+const vec2 = require('gl-matrix').vec2;
```

Comment:
(Note to self for future reference:) It looks like `gl-matrix` has [types on npm](https://www.npmjs.com/package/@types/gl-matrix), so it should be possible to use plain `import` statements and get type-checking for these in the future.